guides/source/i18n.md

```diff
@@ -587,13 +587,6 @@ For example, your `config/locales` directory could look like this:
 
 This way, you can separate model and model attribute names from text inside views, and all of this from the "defaults" (e.g. date and time formats). Other stores for the i18n library could provide different means of such separation.
 
-NOTE: The default locale loading mechanism in Rails does not load locale files in nested dictionaries, like we have here. So, for this to work, we must explicitly tell Rails to look further:
```

Comment:
Instead of removing completely, we should retrofit this to show details on how to add out of path locales to load paths?

Reply:
what do you mean exactly?
if you are running an older version of rails, this info is necessary (but it is available in older versions of the docs)..

Reply:
@johannesschobel It would still make sense to document adding locales from other paths, for example: 
```
# config/application.rb
config.i18n.load_path += Dir[Rails.root.join('vendor', 'some_gem', 'locales', '**', '*.{rb,yml}')]
```